DigitalLearningSolutions.Data/Services/PostLearningAssessmentService.cs

```diff
@@ -25,6 +25,13 @@ public PostLearningAssessmentService(IDbConnection connection, ILogger<PostLearn
 
         public PostLearningAssessment? GetPostLearningAssessment(int customisationId, int candidateId, int sectionId)
         {
+            // NextSectionID is the ID of the next section in the course, according to SectionNumber
+            // or null if the last in the course.
+
+            // Find these by making a list of other tutorials in the course, to find other sections (because a section
+            // must contain at least one tutorial), using a similar approach to the one used in the
+            // TutorialContentService. See the SectionContentService for the definition of a valid section.
```

Comment:
This reference to the section content service would probably also be useful to add to the other comments which mention finding other sections (e.g. diagnostic assessment, tutorial)? 🙂

Reply:
Yep, I agree! Added in 7244c04 🙂